app/styles/app/layouts/billing.scss

```diff
@@ -0,0 +1,135 @@
+
+.billing-form{
+	 h3 {
+		color: $oxide-blue;
+		font-weight: 300;
+		font-size: 18px;
+	}
+}
+
+.billing-plans {
+	display: grid;
```

Comment:
I'm sorry for missing it earlier, but `grid` layout has issues with IE (which we need to support), so I would recommend to use `flex` instead.